frontend/src/components/website/adding_master/StepsProgressBar.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+    <VContainer>
+        <VLayout class="progress-panel">
+            <VFlex
+                ma-1
+                xs4
+                sm4
+                md4
+                class="progress-step"
+                :class="{'active': currentStep === 1,'filled': currentStep >= 1}"
```

Comment:
probably better define computed prop like "firstStepClass" or something like, that will return object based on currentStep

same for other step indicators (secondStepClass etc.)